learn.md

```diff
@@ -70,7 +70,7 @@ redirect_from:
    </div>
    <div class="col-sm-12 col-md-4 cBoxContainer">
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Learn the “batteries-included” standard library.</p>
+     <p class="cBoxpara">Learn the “batteries-included” in the standard library APIs.</p>
```

Comment:
Can't we say
```suggestion
     <p class="cBoxpara">Learn the APIs of the “batteries-included” standard library.</p>
```
instead?

It is the standard library itself that is "batteries-included" IMO, and saying "in the standard library APIs" doesn't seem to read right.

Reply:
Saying both "APIs" and "“batteries-included” doesn't sound right, IMO. Therefore, changed to "Learn the “batteries-included” standard library (APIs)."

Reply:
"batteries-included" is like an adjective for the standard library. So, IMO, if "Learn the APIs of the standard library" reads OK, "Learn the APIs of the "batteries-included" standard library" is also OK? WDYT?

The issue I see with the current suggestion with (APIs) at the end is that it seems disconnected.

It's like we're saying learn the standard library and then just adding "(APIs)" at the end.

Reply:
Nope. Saying  "Learn the APIs of the "batteries-included" standard library" distorts the meaning as the included batteries are the APIs themselves.

@Samallama - WDYT?

Reply:
I don't agree. IMO, batteries-included is used here to say that the standard library itself comes with a lot of functionality, so you wouldn't have to look elsewhere.

I believe it is the standard library as a whole that is batteries-included, not just the APIs.

Another suggestion,

"Check out the API docs of the batteries-included standard library."

Reply:
+1 to say ""Check out the API docs of the batteries-included standard library."

@Samallama - Please confirm.

Reply:
+1

Reply:
I guess it should be

```md
Check out the API docs of the “batteries-included” standard library.
```

Reply:
Added that.